### PR TITLE
Adding scripts to create  fmllr transform and combine trans dirs toge…

### PR DESCRIPTION
**Pull request by marvin-nj**
after combine  lats or ali file , we would want to create trans file from  train dirs , and then combine then together.  we use copy_trans_dir.sh scripts to create trans file from clean train dirs,  and  combine_trans_dirs.sh to comobine each trans dirs  together. Note : you can run them like copy_ali_dir.sh  and combine_ali_dirs.sh.

### CODE REVIEW

**Comment by danpovey** [comment]
thanks.  how did you test this?

**Comment by marvin-nj** [comment]
i have  run it ok repatedly  in my local machine ,  of  course , i hope someone others could check it if possible.

File: egs/wsj/s5/steps/combine_trans_dirs.sh [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  fi
+  if $do_trans && [[ ! -f $src/trans.1 ]]; then
+    echo "$0: warning: transform (trans.*) are not present in $src, not" \
+         "combining. please check you files" 
+    exit 1
+  fi
+done
+
+if [ ! -f $dest/ali.1.gz  ] && [ ! -f $dest/lat.1.gz ] ; then 
+    echo "warning: we assume you have combined the ali or lat dirs " \
```
Comment (danpovey) [inline egs/wsj/s5/steps/combine_trans_dirs.sh]: make sure it starts with "$0: warning: ..." so its clear which program generated the message

File: egs/wsj/s5/steps/combine_trans_dirs.sh [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+         "please run combine_ali_dir.sh or combine_lat_dir.sh firstly"
+    exit
```
Comment (danpovey) [inline egs/wsj/s5/steps/combine_trans_dirs.sh]: and this should be `exit 1`

**Comment by marvin-nj** [comment]
The code looks better Standard now.

**Pull request merged**